config/auspice_config_h5n1.json

```diff
@@ -65,6 +65,11 @@
       "title": "GISAID Clade",
       "type": "categorical"
     },
+    {
+      "key": "clade",
+      "title": "Nextclade Clade",
+      "type": "categorical"
+    },
```

Comment:
I've added the Nextclade Clade as a separate coloring so we can do comparisons across clade labels, but maybe we'll remove `h5_label_clade` eventually?  Would love to hear your thoughts here @lmoncla 🙏